how-it-works/00_General/01_overview-of-the-internet-computer.subpage.md

```diff
@@ -0,0 +1,125 @@
+---
+
+# Architecture of the Internet Computer
+
+The technology behind the Internet Computer (IC) has been reconceived from ground up, learning lessons from the weaknesses of earlier projects, e.g., the lack of scalability.
+The Internet Computer intends to become the World Computer, an open and secure blockchain-based network that can scale out to power humanity's computation.
+
+## Subnet Architecture
+
+The core part of the IC's architecture is its subnet architecture: The IC consists of many subnets, where each subnet is its own blockchain that operates concurrently with all the other subnets.
+New subnets can be added to scale the IC out, analogous to how public clouds scale out.
+Each subnet hosts smart contracts, called *canister smart contracts*, or simply *canisters*.
+Canisters on one subnet can send messages to canisters on the same or other subnet.
+The secure asynchronous cross-subnet (xnet) communication between canisters and the resulting loose coupling of subnets is one of the key principles that enable the scalability of the IC by means of adding new subnets.
+
+<figure>
+<img src="/img/how-it-works/subnet_architecture.png" alt="Architecture: The IC is composed of subnets, each of which is an independent blockchain" title="The IC is composed of subnets, each of which is an independent blockchain" align="center" style="width:600px">
+<figcaption align="center">
+The IC is composed of subnets, each of which is an independent blockchain.<br>
+Canisters within and across subnets communicate through asynchronous messaging.
+</figcaption>
+</figure>
+
+## Canister Smart Contracts
+
+Each subnet can host tens of thousands of *canister smart contracs* or simply *canisters*.
+A canister smart contract is a bundle comprising the smart contract Wasm bytecode and its storage.
+Canister smart contracts can be updated by their maintainer, which can be a single developer, a groups of developers, or a decentralized autonomous organization (DAO).
+For reasons of decentralization, it is best if a canister smart contract be maintained by a DAO in the long run.
+
+Canisters pay for the IC resources they consume in *cycles*.
+To this end, canisters need to be topped up with cycles by their maintainer or anyone else who has an interest in the canister running.
+Cycles can be acquired with the ICP token, the IC's utility token.
+One Trillion cycles can be acquired with ICP worth 1 XDR, where an XDR is a basket comprising major currencies and one XDR is roughly 1.3 USD as of Q3 2022.
+
+Canisters can hold gigabytes of memory for a low fee.
+They can serve the user interface of a dApp directly to a web browser, unlike the UI being served from public cloud as is the case in other blockchains.
+Canisters can be updated and evolve, much like regular software.
+DAO-based governance can make such upgrade process secure and decentralized.
+Using Internet Identity, canisters can authenticate users based on biometry using the WebAuthn protocol.
```

Comment:
Rather than using biometry (which scares some people), I'd rather highlight that we use web authentication to use the secure storage chips built into today's devices.